integration_tests/replica_settings.py

```diff
@@ -60,4 +60,8 @@
     'url': os.getenv('CQRS_BROKER_URL'),
     'consumer_prefetch_count': 2,
     'queue': 'replica',
+    'max_retries': 2,
```

Comment:
Just spotted, sorry. Recently I introduced such structure in settings.
```
settings.CQRS = {
    ...
    'master': {
        'CQRS_AUTO_UPDATE_FIELDS': True,
    },
    ...
}
```

I would strongly suggest to keep to that idea of separation of `master/replica` for all new settings. In other words, could you think of moving new settings to `replica` subobject.

Reply:
Another thing here: `CQRS_AUTO_UPDATE_FIELDS` is written as a Python style constant.
This is done as eventually this thing could be done per Replica model, so it will be defined in code, so please, think if you have such constants and if it makes sense to make them uppercase as well. I definetly don't think that it's required as we can always do 'upper', but I had to note that :)

Reply:
Looks a little bit confusing, i see potential problems for library user.
Before we had simple flat structure in settings, it's easier to understand and harder to misconfigure.

What problem do we solve by doing this separation?

Reply:
Code also will get more complexity.
Example with Infinite (None) settings.

Before
```python
    message_ttl = settings.CQRS.get('message_ttl', DEFAULT_MESSAGE_TTL)
    if message_ttl is None:
        # Infinite
        return
```

After
```python
    master_settings = settings.CQRS.get('master', {})
    if 'message_ttl' in master_settings and master_settings['message_ttl'] is None:
        # Infinite
        return
    
    message_ttl = master_settings.get('message_ttl', DEFAULT_MESSAGE_TTL)
```

Reply:
You got the concept right: maybe it makes sense to support all in lowercase for settings, though.

For me it's easier to understand and I don't see any problems at all (code doesn't count as it's super simple) and, but I see benefits: separation of specific set ups for master and replica. I have to say that in Django world it's a usual practice in settings as well: DATABASES, LOGGING, REST_FRAMEWORK, etc.

@d3rky need your 3rd opinion, as @net-free requested.